src/main/java/org/mule/extension/http/internal/request/HttpRequesterProvider.java

```diff
@@ -106,6 +109,8 @@
 
   private TlsContextFactoryBuilder defaultTlsContextFactoryBuilder = TlsContextFactory.builder();
 
+  private boolean streamResponse = parseBoolean(getProperty(STREAM_RESPONSE_PROPERTY, "true"));
```

Comment:
Isn't it best to define the property such that non-existence of property is same as false anyway? i.e. `mule.http.disableResponseStreaming`